portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,86 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Samiksha Kapoor</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <div class='Content'>
+    <div class='Display'>
+      <img class='ProfilePic' src='images/ProfilePic.jpg' alt='Profilepicture'>
+      <div class="Info">
+        <h1>Samiksha Kapoor</h1>
+        <p>
+          <h2>HEY!</h2>
+          Thanks for viewing my portfolio! Let me introduce myself...
+          <br>
+          I am a Competitive Programmer. I always remain very passionate to learn about
+          new technologies. Loves to solve the Real World Problems and want make a positive impact
+          on this world.I have been a part of lot of communities like Girlscript Patiala, Developer Student Club.
+          Along with that, I am a Microsoft Student partner and a part of WomenTechNetwork.
+          <br>
+          Also, I am a big MUSIC LOVER.
+        </p>
+      </div>
     </div>
-  </body>
+    <p class='GreetingMessage'>Click here to get a message :)</p>
+    <button class="ButtonGreetings" onclick="showRandomQuotes()">Click Here</button>
+    <div class='GreetingMessage' id='greeting-container'></div>
+    <br>
+    <div class='Education'>
+      <h2 class='Headings'> EDUCATION </h2>
+      <ul>
+        <li><b>Thapar Institute of Engineering and Technology,Patiala,Punjab</b></li>
+        <i>2017-2021</i>
+        <br>
+          Bachelors of Engineering (BE) in Computers
+        <li><b>DAV Public School,Amritsar,Punjab</b></li>
+        <i>2017</i>
+        <br>
+          Senior Secondary
+        <li><b>DAV Public School,Amritsar,Punjab</b></li>
+        <i>2015</i><br>
+          Secondary
+      </ul>
+    </div>
+    <h3><b>Check out Some of my work :)</b></h3>
+    <div class='Projects'>
+      <h2 class='Headings'>PROJECTS</h2>
+      <ul>
+        <li><a href="https://github.com/samii9914/PermissionApp.git"><b>GIRLS PERMISSION APP</b></a></li>
```

Comment:
A few places used single quotes but here you have double quotes? You should always use double quotes in HTML and single quotes in JS.